languages/ruby/reference/exercise-concepts/resistor-color-duo.md

```diff
@@ -0,0 +1,54 @@
+# Concepts of resistor-color-duo
+
+## General
+
+ [Example solution](https://github.com/exercism/ruby/blob/master/exercises/resistor-color-duo/.meta/solutions/resistor_color_duo.rb)
+
+- Classes: 
+   - Defining classes
+- Methods: 
+    - Defining methods
+    - Arguments/Parameters
+    - Return values
+- Strings: 
+    - Basics like difference between `1` and `"1"`
+
+- Class methods: 
+    - How to define a method so it can be called directly on a class
+- Constants: 
+    - How to create/declare a constant
+    - Appropriate name for a constant
+    - When should constants be used
+    - Benefits of storing immutable data in a constant
+
+Depending on the approach either:
+- Hashes: 
```

Comment:
Remove all non-semantic trailing white space. There are occurrences in other areas as well.